railgun/keps/0004-kong-kubernetes-testing.md

```diff
@@ -30,15 +43,26 @@ Historically the [Kong Kubernetes Ingress Controller (KIC)][kic] used bash scrip
 - we want our integration tests to be in [Golang][go] to match our primary development language and provide high levels of expressibility for tests
 - we want to be prescriptive how to test Kong Kubernetes components across all [@kong/team-k8s repos][team-repos] to limit to promote code re-use and limit test maintenance
 - we want contributions of integration tests for Kong Kubernetes components to be a friendly experience
-- we want to be able to define cluster environments wholly or in part by a simple container image and tag as the definition
 
 [go]:https://golang.org
 [team-repos]:https://github.com/orgs/Kong/teams/team-k8s/repositories
 
 ### Goals
 
+### Non-Goals
+
+Due to incongruencies with one of our most prominent upstream tools ([Kind][kind]) we're going to need to skip on creating complete container images for testing environments in favor of writing setup logic aftermarket for existing default images. While being able to move runtime problems to build time would be helpful, we'll potentially need to look at migrating to new tools in some future iteration to follow up.
+
+For this iteration we're _not trying to solve the problem of automated cleanup of testing environments_, as this is expected to greatly increase scope. Instead tests are expected to use separate namespaces and cleanup after themselves for this iteration.
```

Comment:
Does this align with https://github.com/Kong/kubernetes-testing-framework/issues/4 ? That suggests we do intend to handle test cleanup for 0.1.0.

Reply:
Yes, the intention is to do it, but not to do it as part of this effort, let's clarify that here:

```suggestion
For this iteration we're _not trying to solve the problem of automated cleanup of testing environments_, as this is expected to greatly increase scope. Instead tests are expected to use separate namespaces and cleanup after themselves for this iteration. There is a follow up item to take care of cleanup as a separate scope: https://github.com/Kong/kubernetes-testing-framework/issues/4.
```